Complete crud functions in AppUserService.java

CemalettinCelalToy opened the pull request:
delete user, update user, and create user functions  have been added

CemalettinCelalToy committed:
Complete crud functions in AppUserService.java

delete user, update user, and create user functions  have been added

shintalha commented on src/main/java/com/beed/service/AppUserService.java:
```diff
@@ -4,41 +4,55 @@
+                    , userDto.getRate(), userDto.getMail(), userDto.getPhone(), userDto.getId());
+        }
+        catch (Exception ex){
+            throw new Exception(ex.getMessage());
```
Kanka burada catchleyip tekrar throw edilmiş, try catchleri controller katmanında yaparz diye düşünmüştüm, o yüzden methodun yanında throws Exception yazıp metodun içinde dümdüz işini yapabilirsin

shintalha replied:
Sanırım diğerlerinde de aynısı olmuş, onları düzeltebiliriz istersen, burada şimdilik try catch koymayalım ben de koymadım

shintalha reviewed (changes requested):
servis katmanındaki try catchlerde exception yakalanıp tekrar throw edilmiş, onlar silinebilir, şimdilik servis katmanında try catch yapmayalımdiye düşündüm, controller kısmında yaparız, ben örneğini yaptım, kodu pushlayınca görünebilecek

CemalettinCelalToy committed:
Update CRUD functions

Role column has been added to functions.
Salt column has been removed from functions.
Exception errors fixed.

mertcarabaci commented:
LGTM!

shintalha reviewed (approved):
Eline sağlık :)
